go.mod

```diff
@@ -8,20 +8,20 @@ require (
 	github.com/getsentry/raven-go v0.2.1-0.20190513200303-c977f96e1095 // indirect
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-test/deep v1.0.5
-	github.com/openshift/api v0.0.0-20200716143919-d16e9583dec6
+	github.com/openshift/api v0.0.0-20200722170803-0ba2c3658da6
 	github.com/openshift/build-machinery-go v0.0.0-20200713135615-1f43d26dccc7
-	github.com/openshift/client-go v0.0.0-20200521150516-05eb9880269c
-	github.com/openshift/library-go v0.0.0-20200709151624-cc498c355c99
+	github.com/openshift/client-go v0.0.0-20200722173614-5a1b0aaeff15
+	github.com/openshift/library-go v0.0.0-20200724150143-bb5c6a560f96
 	github.com/pkg/profile v1.4.0 // indirect
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	go.etcd.io/etcd v0.5.0-alpha.5.0.20191023171146-3cf2f69b5738 // indirect
-	golang.org/x/text v0.3.3 // indirect
-	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/api v0.18.3
-	k8s.io/apimachinery v0.18.3
-	k8s.io/client-go v0.18.3
-	k8s.io/component-base v0.18.3
+	gopkg.in/yaml.v2 v2.3.0
+	k8s.io/api v0.19.0-rc.2
+	k8s.io/apimachinery v0.19.0-rc.2
+	k8s.io/client-go v0.19.0-rc.2
+	k8s.io/component-base v0.19.0-rc.2
 	k8s.io/klog v1.0.0
 	monis.app/go v0.0.0-20190702030534-c65526068664
 )
+
+replace github.com/openshift/api => github.com/pedjak/openshift-api v0.0.0-20200724175332-5af197adc036
```

Comment:
Final commit should not have this line - we want to test early the image building